tests/test_sampling_rates.py

```diff
@@ -66,7 +66,6 @@ def setup_sampling_rates(self):
         context.library > "nodejs@3.14.1" and context.library < "nodejs@4.8.0",
         reason="_sampling_priority_v1 is missing",
     )
-    @bug(library="nodejs", reason="Unexpected amount of sampled traces")
```

Comment:
would be great to know when this was fixed to put a filter instead of removing the whole skip

Reply:
what do you mean by "to put a filter"?

Reply:
Look at the line just above, something like this:
`@bug(context.library < "nodejs@1.2.3"`)

Reply:
Our current goal is to get as many tests to run on dd-trace-js as possible, so don't have time to dig into code archeology at the moment. Would it be okay to leave it for now?

Reply:
+1 @simon-id having an "at least" version is needed. Without that, we won't be able later to run system-tests on old versions. So far, I never dig to find the real version, I always used the current prod version.